tools/downloader/downloader.py

```diff
@@ -317,7 +317,8 @@ def main():
     parser.add_argument('--all', action='store_true', help='download all available models')
     parser.add_argument('--print_all', action='store_true', help='print all available models')
     parser.add_argument('--precisions', metavar='PREC[,PREC...]',
-                        help='download only models with the specified precisions (actual for DLDT networks)')
+                        help='download only models with the specified precisions (actual for DLDT networks): '
```

Comment:
I think the wording should be a bit clearer. For example, "download only models [...]; one or more of: [list of precisions]".